package.json

```diff
@@ -32,12 +32,12 @@
   },
   "homepage": "https://github.com/fix/ark-desktop#readme",
   "devDependencies": {
-    "electron-prebuilt": "^1.4.13",
```

Comment:
Yes, renaming because of deprecation notice from the github page
> Note As of version 1.3.1, this package is published to npm under two names: electron and electron-prebuilt. You can currently use either name, but electron is recommended, as the electron-prebuilt name is deprecated, and will only be published until the end of 2016.